### DISCUSSION TITLE
Multiple PLC read simultaneously

### DISCUSSION THREAD

**Discussion by AlexEspert** [Engineering-Research-and-Development/iotagent-opcua#115] [Q&A]
Good morning Manfredi,

We have been able to connect this github repo (changing the necessary on the config.js) to read one of our PLCs. We would like to connect another PLC using the same IoTAgent. Is it possible?

Thank you in advance,
Alejandro Espert.

**Comment by manfredipist** [comment]
Hello @AlexEspert ! 
Sorry for the delay but i just came back from holidays. We currently do not support multi-server connection with the IoTAgent thus you will need to istantiate multiple agents for each server you desire to connect to.
